lndr-backend/data/lndr-server.config.test

```diff
@@ -35,3 +35,11 @@ heartbeatInterval = 5
 awsPhotoBucket = "lndr-avatars"
 accessKeyId = "AAAAAAAAAAAAAAAAAAAA"
 secretAccessKey= "6444444444444444444444444444444444444444"
+
+# postgres config
+postgresHost = "localhost"
+postgresPort = 5432
+
+# ethereum node config
+web3Host = "http://localhost"
```

Comment:
The value of `web3Host` here appears to be a full URL. We have a couple of options here:

 1. Rename this to `web3Url` and incorporate the port as well, like `http://localhost:8545`, or
 2. Change the expected value to be, like `postgresHost`, a hostname or IP.

If we go the route of option 2, then we may also want a `web3Protocol` attribute that can be either `http` or `https`.

Reply:
Good suggestion. Going with option 1.